doc/classes/Area.xml

```diff
@@ -88,7 +88,7 @@
 			<return type="Array">
 			</return>
 			<description>
-				Returns a list of intersecting [code]Area[/code]s.
```

Comment:
That's actually the doc team's policy to use `[code]` for references to the class being referenced, to avoid creating hyperlinks to itself everywhere. So that's not the right fix - you should do something like b30208439 for `[code]` too.

Reply:
Ah I now see that you've added the proper fix the makerst.py already; so this change (and the same in Area2D) just need to be reverted.